sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/EmptyNullableValueAccess.cs

```diff
@@ -174,5 +141,26 @@ public override IEnumerable<ProgramState> TrySetConstraint(SymbolicValueConstrai
                 return MemberExpression.TrySetConstraint(nullabilityConstraint, programState);
             }
         }
+
+        private sealed class AnalysisContext : ISymbolicExecutionAnalysisContext
+        {
+            private readonly HashSet<IdentifierNameSyntax> nullIdentifiers = new HashSet<IdentifierNameSyntax>();
+            private readonly NullValueAccessedCheck nullPointerCheck;
+
+            public IEnumerable<Diagnostic> GetDiagnostics() =>
+                this.nullIdentifiers.Select(nullIdentifier => Diagnostic.Create(rule, nullIdentifier.Parent.GetLocation(), nullIdentifier.Identifier.ValueText));
+
+            public AnalysisContext(CSharpExplodedGraph explodedGraph)
+            {
+                this.nullPointerCheck = new NullValueAccessedCheck(explodedGraph);
```

Comment:
nitpick: do you need `this` here?

Reply:
No. I forgot to rebase and remove these after #3350 was merged.